src/components/AttentionBox/__tests__/attentionBox-snapshot-tests.jest.js

```diff
@@ -0,0 +1,46 @@
+import React from "react";
+import renderer from "react-test-renderer";
+import { ATTENTION_BOX_TYPES } from "../AttentionBoxConstants";
+import AttentionBox from "../AttentionBox";
+
+describe("AttentionBox renders correctly", () => {
```

Comment:
please add checks for the following props: onClose (because it's what case the close button to be displayed in the attention box), className and creation of attention box with icon type font.